packages/@okta/i18n/src/properties/login.properties

```diff
@@ -844,7 +847,7 @@ oie.numberchallenge.warning = Haven't received a push notification yet? Try open
 ## {0} refers to a number which is the correct answer for the number challenge
 oie.numberchallenge.instruction = On your mobile device, open the Okta Verify prompt, then tap <$1>{0}</$1> in Okta Verify to continue.
 oie.numberchallenge.force.upgrade.title = Update Okta Verify
-idx.authenticator.app.method.push.force.upgrade.number_challenge = Your response was received, but your Okta Verify version is no longer supported by your organization. To verify your identity with push notifications, update Okta Verify to the latest version, then try again.
+oie.authenticator.app.method.push.force.upgrade.number_challenge = Your response was received, but your Okta Verify version is no longer supported by your organization. To verify your identity with push notifications, update Okta Verify to the latest version, then try again.
```

Comment:
Updated key to have `oie` prefix to be consistent with other keys  https://oktainc.atlassian.net/browse/OKTA-381041

Reply:
thanks :)